Week3/homework/js-exercises/1-you-are-amazing.js

```diff
@@ -0,0 +1,28 @@
+{
+    "use strict";
+
+    /*     Write a function named giveCompliment
+    It takes 1 argument: your name
+    Inside the function create an array with 10 strings. Each string should be a compliment, like "great", "awesome"
+    Write logic that randomly selects a compliment
+    Return a string: "You are [COMPLIMENT], [YOUR_NAME]!
+    Call the function three times, giving each function call the same argument: your name.
+     */
+
+    function giveCompliments(name) {
+
+
+
+        let complimentArray = ["Great", "Avesome", "Unique", "Elegant", "Creative", "Smart", "Brave", "Kind", "Funny", "Trustworthy"];
+        let getRandomNumber = [Math.floor(Math.random() * complimentArray.length)];
```

Comment:
nit 1: you do not need the square brackets, the following is clearer: `let getRandomNumber = Math.floor(Math.random() * complimentArray.length);`
nit 2: naming this `randomNumber` is clearer, usually starting a name with a verb (e.g. `get`) is used for naming functions.